src/cborencoder.c

```diff
@@ -227,14 +227,14 @@ static inline bool isOomError(CborError err)
 
 static inline void put32(void *where, uint32_t v)
 {
-    v = cbor_htonl(v);
-    memcpy(where, &v, sizeof(v));
+    uint32_t v_be = cbor_htonl(v);
+    memcpy(where, &v_be, sizeof(v_be));
 }
 
 static inline void put64(void *where, uint64_t v)
 {
-    v = cbor_htonll(v);
-    memcpy(where, &v, sizeof(v));
+    uint64_t v_be = cbor_htonll(v_be);
```

Comment:
Should be `cbor_htonll(v)`

Reply:
Ah yes, sorry about that. Had the right patch in QtBase, but I didn't apply it correctly using the github web ui editor.